composer.json

```diff
@@ -14,13 +14,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.6"
```

Comment:
a minimum requirement should be technical, if the code runs fine on 5.3, that is the minimum required version.